README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# Brief Biography
+#### (hw01)
+
+### **Hello!**
+My name is **Yuanhe Liu**. I am a **MAPSS** student with a concentration in **Psychology**. 
+
+* ### *Some Facts About Me*
+  + I am 22 years old
+  + I came from China
+  + I earned my B.S. degree in Psychology at the University of Washington
+  + My research interests are Clinical Psychology and how cultural factors interact with that
+  
+* ### *Facebook Page*
+  + [Facebook](https://www.facebook.com/profile.php?id=100005223360681)
+
+* ### *A Picture of Me*
+  + ![Picture of Me](hw01.jpeg)
```

Comment:
This picture unfortunately does not appear in the file. The code is right, but you need to use an image linked at a URL rather than a file name like this one. I used the image address of my Facebook profile picture - maybe you could do that as well?